<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="markdown" source-language="en-US" target-language="es-es">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-317e149" tool-company="Microsoft" />
      <xliffext:olfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">bd7cba635c93de4751058aba7b65c1eff2217e2e</xliffext:olfilehash>
      <xliffext:olfilepath xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">docs\visual-basic\misc\bc42031.md</xliffext:olfilepath>
      <xliffext:oltranslationpriority xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">vbcs_lo-mt</xliffext:oltranslationpriority>
      <xliffext:oltranslationtype xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">Machine Translation</xliffext:oltranslationtype>
      <xliffext:olskeletonhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02da59eeea18790d8d43ea1a1f842c5fb5b37e94</xliffext:olskeletonhash>
      <xliffext:olxliffhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">88f59cd13f226a3d687b2f970278d873c7934413</xliffext:olxliffhash>
      <xliffext:poisonError xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">Handback file name: lo-mt_bc42031.d32e47d90c781ad66855a9d24732e64b9170db5f.es-es is different with handoff file name: bc42031.d32e47d90c781ad66855a9d24732e64b9170db5f.es-es.</xliffext:poisonError>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve">
          <source>&amp;#39;Catch&amp;#39; block never reached; <ph id="ph1">&amp;lt;</ph>exception<ph id="ph2">&amp;gt;</ph> handled above in the same &amp;#39;Try&amp;#39; statement | Microsoft Docs</source>
          <target state="translated">Nunca se alcanza; el bloque 'Catch' <ph id="ph1">&amp;lt;</ph>excepción<ph id="ph2">&amp;gt;</ph> controla anteriormente en la misma instrucción 'Try' | Documentos de Microsoft</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve">
          <source>&amp;#39;Catch&amp;#39; block never reached; <ph id="ph1">&amp;lt;</ph>exception<ph id="ph2">&amp;gt;</ph> handled above in the same &amp;#39;Try&amp;#39; statement</source>
          <target state="translated">Nunca se alcanza; el bloque 'Catch' <ph id="ph1">&amp;lt;</ph>excepción<ph id="ph2">&amp;gt;</ph> controla anteriormente en la misma instrucción 'Try'</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve">
          <source>A <ph id="ph1">`Catch`</ph> block in the code cannot be reached because it is handled in a preceding <ph id="ph2">`Try`</ph> block.</source>
          <target state="translated">Un bloque <ph id="ph1">`Catch`</ph> del código no se puede alcanzar porque se controla en un bloque <ph id="ph2">`Try`</ph> precedente.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve">
          <source>By default, this message is a warning.</source>
          <target state="translated">De forma predeterminada, este mensaje es una advertencia.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve">
          <source>For more information about hiding warnings or treating warnings as errors, please see <bpt id="p1">[</bpt>Configuring Warnings in Visual Basic<ept id="p1">](https://docs.microsoft.com/visualstudio/ide/configuring-warnings-in-visual-basic)</ept></source>
          <target state="translated">Para obtener más información acerca de cómo ocultar las advertencias o tratar las advertencias como errores, vea <bpt id="p1">[</bpt>configurar advertencias en Visual Basic<ept id="p1">](https://docs.microsoft.com/visualstudio/ide/configuring-warnings-in-visual-basic)</ept></target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve">
          <source><bpt id="p1">**</bpt>Error ID:<ept id="p1">**</ept> BC42031</source>
          <target state="translated"><bpt id="p1">**</bpt>Identificador de error:<ept id="p1">**</ept> BC42031</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve">
          <source>To correct this error</source>
          <target state="translated">Para corregir este error</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve">
          <source>Remove the redundant statement.</source>
          <target state="translated">Quite la instrucción redundante.</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve">
          <source>See Also</source>
          <target state="translated">Vea también</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve">
          <source><bpt id="p1">[</bpt>How to: Catch an Exception in Visual Basic<ept id="p1">](http://msdn.microsoft.com/en-us/f3063c89-d2bf-49b1-91b5-b87edfb18b95)</ept><ph id="ph1"> </ph></source>
          <target state="translated"><bpt id="p1">[</bpt>Cómo: detectar una excepción en Visual Basic<ept id="p1">](http://msdn.microsoft.com/en-us/f3063c89-d2bf-49b1-91b5-b87edfb18b95)</ept><ph id="ph1"> </ph></target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve">
          <source><bpt id="p1"> [</bpt>How to: Test Code with a Try…Catch Block in Visual Basic<ept id="p1">](http://msdn.microsoft.com/en-us/8368e205-ed73-4185-a247-af84fb4fafa9)</ept><ph id="ph1"> </ph></source>
          <target state="translated"><bpt id="p1"> [</bpt>Cómo: probar el código con un bloque Try... El bloque catch en Visual Basic<ept id="p1">](http://msdn.microsoft.com/en-us/8368e205-ed73-4185-a247-af84fb4fafa9)</ept><ph id="ph1"> </ph></target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve">
          <source><bpt id="p1"> [</bpt>How to: Filter Errors in a Catch Block in Visual Basic<ept id="p1">](http://msdn.microsoft.com/en-us/85964d0a-56e7-4301-a96e-5eaea23b7b9b)</ept><ph id="ph1"> </ph></source>
          <target state="translated"><bpt id="p1"> [</bpt>Cómo: filtrar errores en un bloque Catch en Visual Basic<ept id="p1">](http://msdn.microsoft.com/en-us/85964d0a-56e7-4301-a96e-5eaea23b7b9b)</ept><ph id="ph1"> </ph></target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve">
          <source><bpt id="p1"> [</bpt>Walkthrough: Structured Exception Handling (Visual Basic)<ept id="p1">](http://msdn.microsoft.com/en-us/440da655-4b32-490b-8b16-bfe46f41fa76)</ept><ph id="ph1"> </ph></source>
          <target state="translated"><bpt id="p1"> [</bpt>Tutorial: Control estructurado de excepciones (Visual Basic)<ept id="p1">](http://msdn.microsoft.com/en-us/440da655-4b32-490b-8b16-bfe46f41fa76)</ept><ph id="ph1"> </ph></target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve">
          <source><bpt id="p1"> [</bpt>Try...Catch...Finally Statement<ept id="p1">](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)</ept></source>
          <target state="translated"><bpt id="p1"> [</bpt>Try...Catch...Finally (instrucción)<ept id="p1">](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)</ept></target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>